devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -237,14 +245,33 @@ def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null)
 	changeSwitch(ep, cmd)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicSet cmd, ep = null) { 
+	log.debug "Basic ${cmd}" + (ep ? " from endpoint $ep" : "")
+	[
+		changeSwitch(ep, cmd), 
+		response([
+			"delay 2000",
+			 encap(zwave.meterV3.meterGet(scale: 2), endpoint)
+		 ])
+	]
+}
+
 def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd, ep = null) {
 	log.debug "Binary ${cmd}" + (ep ? " from endpoint $ep" : "")
 	changeSwitch(ep, cmd)
 }
 
+def endpointPollValue() {
```

Comment:
I know I told you otherwise, but now realizing that it is also used in `onOffCmd`, please rename it to something like `defaultEndpoint()`, sorry for the mess.